static/_locales/zh_TW/messages.json

```
@@ -1,8 +1,8 @@
-{
+﻿{
   "appName": {
-    "message": "Global Speed: 視頻速度控制"
+    "message": "Global Speed: 影片速度控制"
   },
   "appDesc": {
-    "message": "設置視頻和音頻的默認速度。"
+    "message": "設定影片和音訊的預設速度。"
```

This could be more fluent!
```suggestion
    "message": "設定影片與音訊的預設播放速度。"
```